config/settings.yml

```diff
@@ -44,7 +44,13 @@ binaries:
 db_encryption_key: f01ff8ebd1a2b053ad697ae1f0d86adb
 
 database_url: postgis:///vets-api
-test_database_url: postgis:///vets-api-test
+
+# environmen variables get stripped when running parallel tests so we set the db here
+test_database_url: <%= if ENV['CI'] || ENV['TEST_ENV_NUMBER']
+                   "postgis://#{ENV['POSTGRES_USER'] || 'postgres'}:#{ENV['POSTGRES_PASSWORD'] || 'password'}@#{ENV['POSTGRES_HOST'] || 'postgres'}:#{ENV['POSTGRES_PORT'] || '5432'}/#{ENV['POSTGRES_DATABASE]'] || 'vets_api_test'}#{ENV['TEST_ENV_NUMBER']}?pool=4"
```

Comment:
😱 

I sat staring at this for 5 minutes. It's basically generating the db url dynamically based on which ENV vars you have set and replacing them with known default values?

Reply:
Yeah, I know it's bad and I hate it too, but I was struggling big time to make anything work.  
Prior to this PR, the value of `test_database_url` in `config/settings.yml` was [overridden by an environment variable set in docker](https://github.com/department-of-veterans-affairs/vets-api/blob/master/docker-compose.test.yml#L20) (which I just learned was a thing from the `config` gem).  But that value seems to be strippped when I run the rake test to run parallel tests.  

Totally open to suggestions on what is less awkward.